### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by estebangarcia21**
I would like to update the README!

### CODE REVIEW

**Comment by Ohmrrr** [comment]
very nice

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # test-repository
-This is my cool repository!
+The repository where we learn about Github.
```
Comment (estebangarcia21) [inline README.md:2]: This is a good change! 👍🏼

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
 # test-repository
-This is my cool repository!
```
Comment (Ohmrrr) [inline README.md:2]: Gonna miss this line 👎

**Review by estebangarcia21 — COMMENTED** [review]
LGTM!

**Pull request merged**